BookingQueueSubscriber/BookingQueueSubscriber.Services/VideoWeb/IVideoWebService.cs

```diff
@@ -11,5 +11,7 @@ public interface IVideoWebService
         Task PushUnlinkedParticipantFromEndpoint(Guid conferenceId, string participantUserName, string jvsEndpointName);
         Task PushLinkedNewParticipantToEndpoint(Guid conferenceId, string participantUserName, string jvsEndpointName);
         Task PushCloseConsultationBetweenEndpointAndParticipant(Guid conferenceId, string participantUserName, string jvsEndpointName);
+        Task PushHearingCancelledMessage(Guid conferenceId);
+        Task PushHearingDetailsUpdatedMessage(Guid conferenceId);
```

Comment:
Would it be worth providing a payload of the latest object?

Reply:
Actually, the video web could probably force get on the conference service, resulting in an up-to-date cache.